.github/workflows/ci.linux.arm.yml

```diff
@@ -34,13 +34,13 @@ jobs:
                          -D PHOTON_ENABLE_URING=ON        \
                          -D PHOTON_ENABLE_LIBCURL=ON      \
                          -D PHOTON_ENABLE_EXTFS=ON
-          cmake --build build -j $(nproc) -- VERBOSE=1
+          cmake --build build -j $(nproc)
```

Comment:
打算精简CI的日志，现在日志过于冗长了。而且包含了一些似乎不应该出现的ERROR，跟DEBUG和INFO混在一起，不够明显。

-----

I'm planning to simplify the CI logs since they are currently too verbose. Additionally, there are some ERROR messages that seem inappropriate, mixed in with DEBUG and INFO, making them less distinct. What do you think?

Reply:
ctest貌似没法传递gflags参数。@ColdWings 知道什么其他办法吗？

或者再增加一个Env

Reply:
@lihuiba 现在情况是CI日志太多可能隐藏问题，比如这个

```
2025-01-06T08:42:42.4161762Z 51: [ RUN      ] ExtfsTest.Unlink
2025-01-06T08:42:42.4162673Z 51: 2025/01/06 08:42:42.414495|ERROR|th=0000000001BBC090|test.cpp:88|lstat:failed lstat [path=/dir1/file3_symlink] errno=2(No such file or directory)
2025-01-06T08:42:42.4164149Z 51: 2025/01/06 08:42:42.414506|ERROR|th=0000000001BBC090|test.cpp:160|unlink:failed unlink [path=/dir1/file6_symlink] errno=2(No such file or directory)
2025-01-06T08:42:42.4165508Z 51: 2025/01/06 08:42:42.414509|ERROR|th=0000000001BBC090|test.cpp:160|unlink:failed unlink [path=/dir1] errno=21(Is a directory)
```

我觉得CI的作用是告诉我们最终是成功了还是失败了，具体怎么失败的可以线下复现。同时通过ERROR日志，打印一些虽然过程异常但是结果没报失败的case，引起关注。